### PR TITLE
docs(Accounts): More Details In Accounting Period

### PR DESCRIPTION
**Pull request by abhishekbalam**
Added points:
- Why accounting period is needed?
- What is the "Closed" option for the selected transactions used for?

Also changed intro.

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
+## 1. Why is Accounting Period needed?
 
-## 1. How to create an Accounting Period
+When transactions are submitted they affect the ledgers and hence the reports which process the ledger data.
+This can cause issues when financial reports have to be generated for audit by authorities or for closing the accounting books for the financial year.
+
+Here Accounting Period can be used to limit the time period within which transactions can be submitted to preserve
+the integrity of the corresponding reports.
+
+## 2. What is the "Closed" option for the selected transactions used for?
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:16]: Please add an image in this section.

**Review by akshitakl — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md (hunk 1 above)
Comment (abhishekbalam) [reply erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:16]: Added 👍

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
 <!-- add-breadcrumbs -->
 # Accounting Period
 
-**An Accounting Period defines a period of time in which financial statements are recorded.**
+**An Accounting Period defines a time period in which financial statements are recorded.**
 
-In ERPNext, defining an Accounting Period freezes the entries for the selected transactions in the selected time frame. This means that the selected transactions will not be submittable in the defined Accounting Period.
+In ERPNext, Accounting Period is a timeframe outside which selected submittable transactions (like Sales/Purchase Invoice, Stock Entry, Payroll Entry, Journal Entry etc) are not allowed to be created. In other words, the selected transactions are only allowed to be created within the Accounting Period.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:6]:
```suggestion
In ERPNext, Accounting Period is a timeframe outside which selected submittable transactions (like Sales/Purchase Invoice, Stock Entry, Payroll Entry, Journal Entry etc) are not allowed to be created. In other words, the selected transactions are only allowed to be created within the defined Accounting Period.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
 
-Consider that you want to generate financial reports for business in Quarter 3 and you want to allow submitting only specific entries in that period. Or you want to freeze all entries that affect the General Ledger. Accounting Period in ERPNext allows doing exactly this.
+## 1. Why is Accounting Period needed?
 
-## 1. How to create an Accounting Period
+When transactions are submitted they affect the ledgers and hence the reports which process the ledger data.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:10]:
```suggestion
When transactions are submitted, they affect the ledgers and the reports which process the ledger data.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md [hunk 4]
```diff
@@ -1,24 +1,39 @@
...
+This can cause issues when financial reports have to be generated for audit by authorities or for closing the accounting books for the financial year.
+
+Here Accounting Period can be used to limit the time period within which transactions can be submitted to preserve
+the integrity of the corresponding reports.
+
+## 2. What is the "Closed" option for the selected transactions used for?
+
+![Accounting Period Child Table](/docs/assets/img/accounts/accounting-period-closed.png)
+
+The "Closed" option in the childtable for transaction doctypes is to select which of them are to be restricted after the end of the Accounting Period.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:20]:
```suggestion
The "Closed" option in the childtable for transaction doctypes is used to select which of them are to be restricted after the end of the Accounting Period.
```

File: erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md [hunk 5]
```diff
@@ -1,24 +1,39 @@
...
+
+Do note that if the Accounting Period ends and if any of the selected transactions in the child table don't have "Closed" checked, then they won't be restricted after the accounting period ends.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/accounting-period.md:22]:
```suggestion
Do note that if the Accounting Period ends and if any of the selected transactions in the child table don't have "Closed" checked, then they won't be restricted after the Accounting Period ends.
```

**Review by prasadarr — CHANGES_REQUESTED** [review]

**Review by akshitakl — APPROVED** [review]

**Pull request merged**